examples/with-vitest/next.config.js

```diff
@@ -0,0 +1,4 @@
+/** @type {import('next').NextConfig} */
+module.exports = {
+  reactStrictMode: true,
+}
```

Comment:
```suggestion
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
}

module.exports = nextConfig
```
This is the current template we are using [here](https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/default/next.config.js) as the type reference doesn't work directly on `module.exports`.